ext/opentelemetry-ext-grpc/src/opentelemetry/ext/grpc/_client.py

```diff
@@ -72,28 +73,45 @@ def callback(response_future):
                 return
             response = response_future.result()
             rpc_info.response = response
+            if "ByteSize" in dir(response):
+                metrics_recorder.record_bytes_in(
+                    response.ByteSize(), client_info.full_method
+                )
 
     return callback
 
 
 class OpenTelemetryClientInterceptor(
     grpcext.UnaryClientInterceptor, grpcext.StreamClientInterceptor
 ):
-    def __init__(self, tracer):
+    def __init__(self, tracer, exporter, interval):
         self._tracer = tracer
 
+        self._meter = None
+        if exporter:
```

Comment:
Will users be able to configure exporters during runtime (like set_Exporter or something), because now without an exporter passed in, the metrics recorder will still record metrics into the given meter. If they are not exported anywhere, is there any point in collecting them at all?

Reply:
If there is no exporter the meter should be None, with nothing recorded to it. The interceptor is not supposed to be exposed to the user (private attribute on channel), so I don't think putting a public set_exporter on that is a good idea